### DISCUSSION TITLE
[CodeStyle] Brackets for methods start on the next line

### DISCUSSION THREAD

**Discussion by ViMaSter** [Studio-Lovelies/GG-JointJustice-Unity#165] [Ideas]
> Brackets for methods start on the next line

---

I'd suggest updating this to `Curly braces for method bodies start on the next line, on the same indentation level, as the method declarations`.
I think brackets describe `[]`, not `()` and `{}`, but this is what is actually being referred to? (Native speakers, feel free to chime in here. : D)

**Comment by PhoebeMitchell** [comment]
Completely misread this first time around if you got an email for my first comment :D

You are correct, it should be braces.
